package.json

```diff
@@ -49,6 +49,7 @@
     "test:lint": "tslint --project . --format codeFrame && tslint '{test,electron,src}/**/*.ts' --format codeFrame && prettier '{*,**/*}.{js,ts,md,json,scss}' --list-different",
     "fix:tslint": "tslint --fix --project . && tslint --fix '{test,electron,src}/**/*.ts'",
     "fix:prettier": "prettier --write '{*,**/*}.{js,ts,md,json,scss}'",
+    "fix": "npm run fix:tslint && npm run fix:prettier",
```

Comment:
Thank you!! ❤️

Reply:
I am trying to add the `fix` target to all bitcore modules.